packages/tools/devtools/devtools-view/src/test/SettingsView.test.tsx

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import { debug } from "jest-preview";
+import React from "react";
+
+import { SettingsView } from "../components/index.js";
+
+import { assertNoAccessibilityViolations } from "./utils/index.js";
+
+describe("SettingsView Accessibility Check", () => {
+	it("SettingsView is accessible", async () => {
+		const { container } = render(<SettingsView />);
+		await assertNoAccessibilityViolations(container);
+	});
+
+	it("Can tab/arrow navigate through the SettingsView", async () => {
+		render(<SettingsView />);
+
+		const user = userEvent.setup();
+		// Focus on the first interactive element (dropdown theme selector)
+		await user.tab();
+		const dropdown = screen.getByRole("combobox", { name: /theme selection dropdown/i });
+		expect(dropdown).toHaveFocus();
+		// The dropdown theme options are divs but get compiled as buttons in the expect function and thus fail any expect calls.
+		await user.keyboard("{Enter}");
```

Comment:
What's this Enter for? I'd imagine it expands the dropdown but then we don't do anything with the dropdown anymore and tab to the next component.

Reply:
Ah I forgot to remove this.